packages/react/src/components/Select/index.test.tsx

```diff
@@ -0,0 +1,263 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { renderToString } from 'react-dom/server'; // convert the React element to HTML
+import Select from './';
+import axe from '../../axe';
+
+const defaultProps = {
+  name: 'Test Select',
+  label: 'Test Select'
+};
+
+const withCustomOptions = (otherProps = {}) => {
+  const options = [
+    { key: '1', value: 'Fred' },
+    { key: '2', value: 'Bill' },
+    { key: '3', value: 'Ted' },
+    { key: '4', value: 'Bob' }
+  ];
+
+  return render(
+    <div>
```

Comment:
Just curious, is the outer `<div>` needed here for any reason? I realize this was converted over from the old test but would be nice to clean it up if it's not needed.